### PR TITLE
Wicket versions are aligned and set to the most recent release

### PR DESCRIPTION
**Pull request by solomax**

### CODE REVIEW

**Comment by klopfdreh** [comment]
Looks good to me.

**Comment by chrismattmann** [comment]
thank you @solomax and @klopfdreh this also LGTM! I'll test it in DRAT shortly (on Proteus) and if it works out I'll commit it shortly!

File: proteus/src/main/java/drat/proteus/rest/DratRestResource.java [hunk 1]
```diff
@@ -17,34 +17,37 @@
...
 
   private static final long serialVersionUID = -5885535059043262485L;
   public AbstractOodtWrapper oodtWrapper;
   public AbstractDratWrapper dratWrapper;
 
   public DratRestResource() {
+    super(new GsonWebSerialDeserial());
     oodtWrapper = ProcessOodtWrapper.getInstance();
     dratWrapper = ProcessDratWrapper.getInstance();
     GsonBuilder g = new GsonBuilder();
```
Comment (chrismattmann) [inline proteus/src/main/java/drat/proteus/rest/DratRestResource.java:49]: should I also remove this line (and the one below) @solomax ?

Comment (solomax) [reply proteus/src/main/java/drat/proteus/rest/DratRestResource.java:49]: I was unable to compile Proteus with tests enabled :(
And not sure how to run it :( standard `mvn jetty:run` not working ...
Can perform some other improvements if you have small how-to on this

Comment (chrismattmann) [reply proteus/src/main/java/drat/proteus/rest/DratRestResource.java:49]: hey @solomax check out this page: https://github.com/chrismattmann/drat/wiki/Installation I normally just run through that process to rebuild Proteus...I don't use Jetty to use it's a Tomcat deployment does mvn tomcat7:run work?

**Comment by chrismattmann** [comment]
I tested this it runs fine:

screen shot 2017-08-03 at 1 21 59 pm

**Comment by chrismattmann** [comment]
I'm going to make one minor update and commit...done in https://github.com/chrismattmann/drat/commit/721148db6db25be5d3c1e77d5aae1e53d1d640c2

**Comment by chrismattmann** [comment]
thanks @solomax !

**Comment by klopfdreh** [comment]
thanks for the detailed feedback @chrismattmann - good to see everything is running fine. 👍

**Comment by chrismattmann** [comment]
it's running great @klopfdreh ! For fun I ran Proteus on Wicket, here ya go!

screen shot 2017-08-03 at 2 01 44 pm

**Comment by chrismattmann** [comment]
mostly text, Java, and HTML and about 50/50 split between ASF licensed code, and Standards (BSD, etc.) licensed :) 👍

**Comment by bitstorm** [comment]
Hi everybody!
@solomax why you turned 'GsonRestResource' into 'AbstractRestResource<GsonWebSerialDeserial>'? I mean, it's ok, I ask it just for curiosity.

**Comment by solomax** [comment]
Hello @bitstorm,
I was unable to find `GsonRestResource` in wicketstuff 7.8.0.
Investigate a little and find out this class was very simple 'specification' of `AbstractRestResource`.
This is why I have asked to review the PR :))

**Comment by bitstorm** [comment]
Ouch! Sorry, now I remember that I've removed it from 7.x :sheep: 

Good work!

**Comment by chrismattmann** [comment]
hi @bitstorm and @solomax and @klopfdreh quick question. Now in the latest PR merge, when there is like a bug, or NPE in the service, instead of getting a stack trace, I get a simple message:

```General server error.```

Do you know how I can find the real message? I'm getting this error when running DRAT Proteus against the Wicket repo itself...there is something messed up in the repository service, and I can't find out where the bug/NPE/error is because it's getting swallowed (whereas before this PR I could see the full stack trace). Any ideas?

**Comment by martin-g** [comment]
Hi @chrismattmann !
Where do you see `General server error.` ? 
In Tomcat logs ?
Does it at least say which class logs this generic message ?
I don't see anything in this PR that would cause such change in the logging behavior.

**Comment by chrismattmann** [comment]
Hi @martin-g thanks. Nothing in the PR per-se that specifically causes an error, just something that kind of masks it. The PR changed the abstract base class and upgraded to a newer version of Wicket. Now, when I visit this URL after running DRAT:

http://localhost:8080/proteus/service/repo/licenses/unapproved

(on Wicket repo, but say, not on DRAT repo) I see 

```General server error.``` and it doesn't really show me anything in the logs. That's the crazy part - is that before when there was an error in a web service (the above REST url is mounted via ServicesRestResource.java), I would at least get a stack trace, and it's now being swallowed. Any ideas I'd appreciate...

**Comment by martin-g** [comment]
Here is the problem: https://github.com/wicketstuff/core/blob/master/wicketstuff-restannotations-parent/restannotations/src/main/java/org/wicketstuff/rest/resource/AbstractRestResource.java#L647
You will need to override this method to log the details.
@bitstorm  IMO the default impl should log the exception as an error. If anyone wants to suppress it then (s)he can override it and do something custom.

**Comment by chrismattmann** [comment]
Thanks @martin-g yes that's the issue and yes I agree, it would be great @bitstorm if you agree that it should log the exception as an error when that happens by default, and then yes, allow override.

**Comment by bitstorm** [comment]
Ok! I will add the log.

**Comment by bitstorm** [comment]
Done https://github.com/wicketstuff/core/commit/5900a524512df879ef97d235efc9ad202f949f88

**Comment by chrismattmann** [comment]
thank you @bitstorm !

**Pull request merged**